package.json

```diff
@@ -230,7 +230,9 @@
     "gridplus-sdk/elliptic": "^6.6.1",
     "@ethereumjs/tx": "patch:@ethereumjs/tx@npm%3A5.4.0#~/.yarn/patches/@ethereumjs-tx-npm-5.4.0-0c4a0f973e.patch",
     "@keystonehq/metamask-airgapped-keyring": "patch:@keystonehq/metamask-airgapped-keyring@npm%3A0.15.2#~/.yarn/patches/@keystonehq-metamask-airgapped-keyring-npm-0.15.2-94dd4b40d7.patch",
-    "@trezor/connect-web": "~9.4.7"
+    "@trezor/connect-web": "~9.4.7",
+    "@metamask/controller-utils@npm:^11.7.1-backport": "11.7.1-backport",
+    "@metamask/snaps-rpc-methods@npm:^12.1.0": "patch:@metamask/snaps-rpc-methods@npm%3A12.1.0#~/.yarn/patches/@metamask-snaps-rpc-methods-npm-12.1.0-052e5af572.patch"
```

Comment:
Introduced in https://github.com/MetaMask/metamask-extension/pull/33202

Reply:
This won't be applied properly because `@metamask/snaps-rpc-methods@npm:^12.1.0` isn't being used here. 12.3.0 is being used.

I'd recommend doing what the `main` version of this did instead, and update to v12.4.0: https://github.com/MetaMask/metamask-extension/pull/33203/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

Either that or create a new patch for v12.3.0.

Thoughts @FrederikBolding @Mrtenz @GuillaumeRx ?

Reply:
cc @mcmire too